src/shared/components/nav-pill/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+
+import styles from './index.module.css'
+
+class NavPill extends PureComponent {
+  render () {
+    const { index, title, active, onPillClick } = this.props
```

Comment:
this seems wrong because the `prop` should be `onClick` instead of `onPillClick`, or the opposite.